cellmlmanip/model.py

```diff
@@ -92,19 +92,24 @@ def add_number(self, value, units):
         Creates and returns a :class:`NumberDummy` to represent a number with units in sympy expressions.
 
         :param number: A number (anything convertible to float).
-        :param units: A string unit representation.
+        :param units: A `pint` units representation
```

Comment:
As this also accepts strings, should we keep that option in the docs too? Or hide it just for use by our tests?

Reply:
Oh, and should it be double backticks? My reST is rusty!